lib/orocos/process.rb

```diff
@@ -902,9 +920,6 @@ def command_line(working_directory: Orocos.default_working_directory,
                         " Valid options are #{valid_levels}."
                 end
             end
-            if name_service_ip
-                result.env['ORBInitRef'] = "NameService=corbaname::#{name_service_ip}"
-            end
```

Comment:
That needs to stay. Even though the syskit instance won't be using the name service anymore, it's still useful for the non-syskit tooling (e.g. rock-display, orops, small quick-and-dirty ruby scripts) and even for the syskit tooling until we update it (e.g. syskit ide)